### PR TITLE
Fix test spec for scheduled jobs

### PR DESCRIPTION
**Pull request by guangy10**

### CODE REVIEW

**Comment by guangy10** [comment]
`pull / test-llama-runner-qnn-linux` is failing. IIUC https://github.com/pytorch/executorch/pull/4662 is backed out, so should this job relying on qnn be disabled as well? @cccclai

File: .github/workflows/android-perf.yml [hunk 1]
```diff
@@ -268,6 +278,6 @@ jobs:
       # TODO: Hard code llm_demo_bpe for now in this job.
       android-app-archive: https://gha-artifacts.s3.amazonaws.com/${{ github.repository }}/${{ github.run_id }}/artifact/llm_demo_bpe/app-debug.apk
       android-test-archive: https://gha-artifacts.s3.amazonaws.com/${{ github.repository }}/${{ github.run_id }}/artifact/llm_demo_bpe/app-debug-androidTest.apk
-      test-spec: ${{ inputs.test_spec }}
+      test-spec: ${test_spec}
```
Comment (huydhn) [inline .github/workflows/android-perf.yml:281]: This syntax looks wrong as GitHub uses something like ${{ env.test_spec }} in their yaml workflow instead (https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables).  I think the fix could be to move the default value here, i.e. https://github.com/pytorch/executorch/pull/4736

Comment (guangy10) [reply .github/workflows/android-perf.yml:281]: Aha, I don't get a chance to really test it yet. If you already got a proper fix, let's go for it.

**Comment by guangy10** [comment]
Merged to https://github.com/pytorch/executorch/pull/4736

**Pull request closed without merging**